kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/log/LogBatchCommand.scala

```diff
@@ -100,7 +138,7 @@ class LogBatchCommand(
     var batch: Batch = null
 
     if (!normalizedCliConfig.logOpts.forward) {
-      return (batch, true)
+      return (batch, log != null)
```

Comment:
would you please explain a little about why this change is required?

Reply:
before, the fetch log and check state in the same context.

Now the fetch log operation is in log thread.